.github/workflows/reuse.yml

```diff
@@ -0,0 +1,254 @@
+name: If Levaithan was reusuable?
+
+on:
+  workflow_call:
+    inputs:
+      LEVIATHAN_REF:  # id of input
+        description: "Leviathan branch, or tag, or commit to test with. Examples: 'refs/heads/<branchName>', 'refs/tags/<tagName>', '<commitId>'"
+        required: false
+        type: string
+        default: 'refs/heads/master'
+      # CONFIG:
+      #   description: 'path to config.js'
+      #   required: false
+      #   default: './suites/config.js'
+      SUITES:
+        description: 'path to test folders'
+        required: false
+        type: string
+        default: './suites'
+      # TEST_SUITE_NAME:
+      #   description: 'which test suite to run'
+      #   required: false
+      #   type: string
+      WORKSPACE:
+        description: 'path to WORKSPACE'
+        required: false
+        type: string
+        default: './.leviathan-inputs'
+      DEVICE_TYPE:
+        description: "slug of device type"
+        required: true
+        type: string
+      REPORTS:
+        description: "path to reports directory"
+        required: false
+        type: string
+        default: "./reports"
+      ENVIRONMENT:
+        description: "What BalenaCloud env, balena-cloud or balena-machine"
+        required: true
+        type: string
+        default: "balena-cloud"
+      BALENACLOUD_ORG:
+        required: false
+        description: "Organization used during cloud suite to create fleets, etc."
+        default: "testbot"
+        type: string
+      BALENACLOUD_APP_NAME:
+        description: "Testbot fleet for finding available Leviathan workers. Not used for QEMU workers. Can accept a list of apps separated by commas."
+        type: string
+        required: false
+        default: "balena/testbot-rig"
+      BALENACLOUD_API_URL:
+        type: string
+        required: false
+        default: "balena-cloud.com"
```

Comment:
How is this different from `ENVIRONMENT` above? We should combine them

Reply:
Can combine, but requires a tinker. In Jenkins we created an env variable called ENVIRONMENT with 2 potential values (balena-cloud, balena-machine). 

This ENVIRONMENT decides all of the other env variables to be used next in the config.js, check leviathan/suites/e2e's config.js